cocos/gfx/device-manager.ts

```diff
@@ -161,7 +161,7 @@ export class DeviceManager {
     }
 
     private _initSwapchain () {
-        const swapchainInfo = new SwapchainInfo(this._canvas!);
+        const swapchainInfo = new SwapchainInfo(0, this._canvas!);
```

Comment:
Web has only one window. Is 0 a valid windowId？ @minggo

Reply:
Valid windowId start from 1.

Reply:
Valid windowId start from 1.